src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx

```diff
@@ -257,24 +258,24 @@ class IntroductionPage extends React.Component {
                   can print this for your records.
                 </p>
               )}
-            </li>
-            <li className="process-step list-three">
-              <h3 className="vads-u-font-size--h4">VA review</h3>
+            </va-process-list-item>
+            <va-process-list-item header="VA review" uswds>
+              {/* VA review */}
               <p>
                 We process applications in the order we receive them. The amount
                 of time it takes to process your claim depends on how many
                 injuries or disabilities you claim and how long it takes us to
                 gather evidence needed to decide your claim.
               </p>
-            </li>
-            <li className="process-step list-four">
-              <h3 className="vads-u-font-size--h4">Decision</h3>
+            </va-process-list-item>
+            <va-process-list-item header="Decision" uswds>
+              {/* Decision */}
               <p>
                 Once we’ve processed your claim, you’ll get a notice in the mail
                 with our decision.
               </p>
-            </li>
-          </ol>
+            </va-process-list-item>
+          </va-process-list>
```

Comment:
Could we add some unit tests? Perhaps a check in src/applications/disability-benefits/all-claims/tests/containers/IntroductionPage.unit.spec.jsx to verify each process list item is on the page?

Reply:
I've updated the unit test in that file to test for each step, instead of just the first step